examples/gltf/package.json

```diff
@@ -10,7 +10,8 @@
   },
   "dependencies": {
     "@loaders.gl/gltf": "^0.7.2",
-    "luma.gl": "^6.2.0-beta"
+    "luma.gl": "^6.2.0-beta",
+    "webvr-polyfill": "^0.10.10"
```

Comment:
(not for this PR) Looked at the code it is pretty small, maybe we can handle this in the framework without a dependency?

Reply:
I think I can make things work without this.